### DISCUSSION TITLE
API for https://github.com/dashboard-feed ?

### DISCUSSION THREAD

**Discussion by mykiwi** [octokit/.github#53] [Q&A]
Hi

I want to know if there is an API to get the same content as display in `https://github.com/dashboard-feed` like following activities, stars, new repo, ...

Thanks

**Comment by gr2m** [comment]
General API questions are best asked here:
https://github.com/orgs/community/discussions/categories/api-and-webhooks

Available APIs are documented here

1. REST API: https://docs.github.com/en/rest
2. GraphQL API: https://docs.github.com/en/graphql
